lib/user.ml

```diff
@@ -0,0 +1,23 @@
+type user = { id : string; name : string; email : string }
+
+let pool =
+  Dotenv.export () |> ignore;
```

Comment:
I'm unsure if dotenv is the recommended way to load .env variables. I've followed [this](https://github.com/thatportugueseguy/ocaml-dotenv?tab=readme-ov-file#usage)